src/main/java/io/cloudchains/app/App.java

```diff
@@ -19,7 +19,7 @@ public class App {
 
 	private static final boolean isLoggingEnabled = false;
 
-	public static String BASE_URL = "https://utils.blocknet.org/";
+	public static String BASE_URL = "http://194.5.152.211:5000/";
```

Comment:
Do we really want to have `"http://194.5.152.211:5000/";` as the endpoint in the master branch?  I thought that endpoint was just for testing...?